apps/store/src/components/Perils/Perils.tsx

```diff
@@ -42,7 +42,7 @@ export const Perils = ({ items }: Props) => {
       {getPerilColumns(items, getColumnCountByBreakpoint(BREAKPOINTS)).map((perils, index) => (
         <PerilColumnFlex key={index}>
           {perils.map((peril) => (
-            <PerilsAccordion key={index} peril={peril} />
+            <PerilsAccordion key={peril.title} peril={peril} />
```

Comment:
Could we do something like:

```suggestion
            <PerilsAccordion key={${index}-${peril.title}} peril={peril} />
```

In the storybook it's possible to have two titles with the same title.

Reply:
Yes, I believe we can do that 👍🏻